### ISSUE TITLE
Add cases of facility to Frontend

### ISSUE THREAD

**Issue by shahraz1998** [amosproj/amos2024ss01-xcelerator-demo-app#250]
## User story

I want to be able to see the amount of cases of a facility and not just a randomly generated number

## Acceptance criteria
- [ ] adjust the amount of open cases of a facility to actually be the amount of open cases from the backend(detail page, overview page)
- [ ] show the first 3 open cases linked to this facility and if you click it you are redirected to that specific case (if there are no or 1/2 of course show "no cases" or the 1/2 links)
- [ ] if there are more than 3 cases show "Go to all Cases" and link to the cases overview
- [ ] if there are exactly 4 cases just show the 4th case instead of "Go to all Cases"

## Definition of done (DoD)

-     Added only after week 5
-     The same for all features
-     Here goes the project specific part

## DoD general criteria

-     Feature has been fully implemented
-     Feature has been merged into the mainline
-     All acceptance criteria were met
-     Product owner approved features
-     All tests are passing
-     Developers agreed to release

**Comment by Sungonogi** [comment]
I decided to also add a button "Go to all Cases" and also moved the new Case to the right

![Image](https://github.com/user-attachments/assets/b5e2f6e1-7a1f-4100-9bd3-3824cdb337ca)


I also decided that if there are 0 cases it should not display a number next to the icon at all


Actually for the situation that there are exactly 4 cases I noticed showing "Go to all Cases" is dumb so in that case I just show the 4th one, for >4 cases it still shows "Go to all cases"


![Image](https://github.com/user-attachments/assets/8c8b1813-f407-4ec6-8362-205b416d8a71)

**Comment by Sungonogi** [comment]
In the code review @Hydraneut pointed out that showing "First/Second/... Case" does not make sense as they don't really have an order. So we adjusted it to show the id of the case

![Image](https://github.com/user-attachments/assets/10d2f82b-97f6-478c-a2c6-487f1155f51d)
